install.sh

```diff
@@ -211,9 +232,15 @@ verify_tarball() {
     fi
 }
 unpack_tarball() {
-    info "unpacking tarball file"
-    mkdir -p /usr/local
-    tar xzf $TMP_TARBALL -C /usr/local
+    info "unpacking tarball file to ${INSTALL_RKE2_TAR_DIR}"
+    mkdir -p ${INSTALL_RKE2_TAR_DIR}
+    tar xzf $TMP_TARBALL -C ${INSTALL_RKE2_TAR_DIR}
+    if [ "${INSTALL_RKE2_TAR_DIR}" != "/usr/local" ]; then
+        info "updating tarball contents to reflect install path"
+        sed -i "s|/usr/local|${INSTALL_RKE2_TAR_DIR}|" ${INSTALL_RKE2_TAR_DIR}/lib/systemd/system/rke2-*.service ${INSTALL_RKE2_TAR_DIR}/bin/rke2-uninstall.sh
+        mv -vf ${INSTALL_RKE2_TAR_DIR}/lib/systemd/system/rke2-*.service /etc/systemd/system/
+        info "you may want to run:  export PATH=\$PATH:${INSTALL_RKE2_TAR_DIR}/bin"
```

Comment:
Can we provide better (i.e. more verbose) information telling the user where the tarball contents are ending up? i.e. between `/etc/systemd/system` and `/opt/rke2`

Reply:
We do that up here at https://github.com/rancher/rke2/pull/743/files#diff-043df5bdbf6639d7a77e1d44c5226fd7371e5259a1e4df3a0dd5d64c30dca44fR235

Reply:
Sure, we say we're untarring to `/opt/rke2` but other than the `.service` files being moved with a `-v` there really isn't much context there to the end user. I think we should more clearly state that we are explicitly placing the unit file in the `/etc/systemd/system` directory as this diverts from existing behavior of the tarball, otherwise a user might assume that the unit file should be in `/opt/rke2/lib/systemd/system` even though systemd doesn't actually search this directory.

Reply:
Fair enough, how's it look now?

Reply:
Looks great!